### DISCUSSION TITLE
Adding support for different hardware backends would be really awesome

### DISCUSSION THREAD

**Discussion by saurabhtangri** [ThanatosShinji/onnx-tool#86] [Ideas]
http://llm-viewer.com/

Counting MACs this quick is great. What would make this tool very useful is to be able to also see roofline.

**Comment by ThanatosShinji** [comment]
it's a good suggestion. Adding the hardware capability from the spec is easy, but not useful. It's difficult to collect the real hardware efficiency for each device. Maybe I can start from the theoretical values.

**Reply by ThanatosShinji** [reply]
it's been added by the recent commit.
